test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -348,14 +348,12 @@ define([
                     channel: jcm.MESSAGE_TYPE.LOGS,
                     message: {
                         [jcm.PARAM.JOB_ID]: TEST_JOB_ID,
-                        first_line: 2000,
-                        latest: true,
+                        [jcm.PARAM.FIRST_LINE]: 2000,
```

Comment:
use the vocabulary terms

I removed the `latest` param since we don't use it ATM